boards/nuvoton/npck3m8k_evb/Kconfig.defconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2025 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config SYS_CLOCK_TICKS_PER_SEC
+	default 1000
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 15000000
```

Comment:
this needs to come from a dts property and using the dts functions, and should be set at SoC level in the Soc Kconfig.defconfig file